docs/library/machine.I2C.rst

```diff
@@ -112,19 +112,21 @@ control over the bus, otherwise the standard methods (see below) can be used.
 
    Availability: ESP8266.
 
-.. method:: I2C.readinto(buf)
+.. method:: I2C.readinto(buf, nack=True)
```

Comment:
I'm slightly confused about this. Only a slave sends NAKs or ACKs. A master NEVER sends ACKs or NAKs, only receives them.